adafruit_requests.py

```diff
@@ -551,6 +558,16 @@ def _send(socket: SocketType, data: bytes):
                 raise OSError(errno.EIO)
             total_sent += sent
 
+    def _send_as_bytes(self, socket: SocketType, data: str):
+        return self._send(socket, bytes(data, "utf-8"))
+
+    def _send_header(self, socket, header, value):
+        self._send_as_bytes(socket, header)
+        self._send_as_bytes(socket, ": ")
```

Comment:
Cheaper, because no run-time call to `bytes()`.
```suggestion
        self._send(socket, b": ")
```

Reply:
fixed in https://github.com/adafruit/Adafruit_CircuitPython_Requests/pull/149/commits/5fa5aa218bbaecdd7b81c1e33a0894de635a8eff